data/mods/Magiclysm/items/enchanted_clothes.json

```diff
@@ -0,0 +1,95 @@
+[
+  {
+    "id": "ench_parabolan_blouse",
+    "type": "enchantment",
+    "has": "WORN",
+    "condition": "ALWAYS",
+    "values": [ { "value": "DEXTERITY", "add": 2 }, { "value": "BONUS_DODGE", "add": 3 }, { "value": "SOCIAL_LIE", "add": 3 } ],
+    "mutations": [ "MIRROR_DISPLACEMENT" ]
+  },
+  {
+    "id": "enchanted_pirate_blouse",
+    "type": "ARMOR",
+    "name": { "str_sp": "enchanted parabolan wool blouse" },
+    "description": "A loose iridescent blouse in a style most often associated with pirates and renfaires.  It shimmers as you look at it.  Made of parabolan wool, it sometimes dimly shows reflections of things here and not.  This particular one has piratical flairs.",
+    "looks_like": "dress_shirt",
+    "weight": "60 g",
+    "volume": "750 ml",
+    "price": 200000,
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "torso", "arm_l", "arm_r" ],
+    "coverage": 90,
+    "encumbrance": 2,
+    "material": [ "parabolan_wool" ],
+    "material_thickness": 0.5,
+    "flags": [ "VARSIZE", "WATER_FRIENDLY", "RAINPROOF", "SUPER_FANCY", "STURDY", "TRADER_KEEP_EQUIPPED" ],
+    "relic_data": { "passive_effects": [ { "id": "ench_parabolan_blouse" } ] }
+  },
+  {
+    "id": "parabolan_enchanted_breeches",
+    "type": "ARMOR",
+    "name": { "str_sp": "enchanted parabolan wool breeches" },
+    "description": "A well-made pair of old-fashioned pants, made of iridescent material.  Comfortable, but lacks pockets.  It shimmers as you look at it.  Made of parabolan wool, it sometimes dimly shows reflections of things here and not.  This particular one has piratical flairs.",
+    "weight": "52 g",
+    "volume": "1750 ml",
+    "price": 490000,
+    "price_postapoc": 50000,
+    "material": [ "parabolan_wool" ],
+    "symbol": "[",
+    "looks_like": "pants",
+    "color": "brown",
+    "covers": [ "leg_l", "leg_r" ],
+    "coverage": 80,
+    "encumbrance": 4,
+    "material_thickness": 0.5,
+    "flags": [ "VARSIZE", "WATER_FRIENDLY", "RAINPROOF", "SUPER_FANCY", "STURDY", "TRADER_KEEP_EQUIPPED" ],
+    "relic_data": {
+      "passive_effects": [
+        {
+          "has": "WORN",
+          "condition": "ALWAYS",
+          "values": [ { "value": "ARMOR_BULLET", "add": 8 }, { "value": "ARMOR_STAB", "add": 3 }, { "value": "ARMOR_CUT", "add": 3 } ]
```

Comment:
The average player would ignore the item, not knowing about effects.
Until we get some way to display those in item's view, maybe add some info to the description?

Reply:
I keep going back and forth on this.   I don't particularly want the player to kill the merchant just for their clothes but at the same time you are right that it looks absolutely normal to the player.

Reply:
I decided not to state what the shirts do but to list that they "glitter with enchantment".